### ISSUE TITLE
Toolbar present on @@manage-viewlets

### ISSUE THREAD

**Issue by hvelarde** [plone/Products.CMFPlone#1557]
Plone 5.0.4 acessing the `@@manage-viewlets` browser view:

![selection_066](https://cloud.githubusercontent.com/assets/341393/14906629/f882e958-0d93-11e6-8b8c-8e85bc90c7b7.png)

**Comment by davilima6** [comment]
The problem isn't the toolbar being present - it should be everywhere, like in `@@manage-portlets`. It's excessive height and broken contrast coming from `.plone-toolbar-top #edit-zone nav>ul` styles. When using default side toolbar we get the same legibility problem.

We could either fix that or blacklist the `plone.toolbar` viewlet manager. It's not very useful for a Site Admin to hide the only `plone.contentviews` viewlet in there: the toolbar would still show, with the small Plone logo for retract/expand and the User Menu - but would lack _all_ of its icons. Besides I think it'll be rare for addons to register viewlets there.

**Comment by ale-rt** [comment]
The `@@manage-viewlets` view is subject to this kind of weird behavior.
In addition it does not allow to handle the viewlets in the had.
But I would say this goes beyond the scope of the issue you reported.

Given the issue is stale since quite some time I will close, but feel free to reopen if you think it should be addressed in the future Plone versions.
